### ISSUE TITLE
Issue when installing with macOS with intel processor

### ISSUE THREAD

**Issue by umarteauowkin** [owkin/FLamby#245]
Running the make install on a mac with macOS 12.6, I had the following problem when running my first strategy (as in the quickstart).

I ran it from pycharm  and got the following error 

OMP: Error #15: Initializing libiomp5.dylib, but found libomp.dylib already initialized.

Apparently, this error is quite classical (see link below)

In that case, the workaround is  to create the conda env, install nomkl and then pip install -e . 

I am not sure this can be smoothly integrated in the make install, but if not could it be specified that this might happen for mac users ?

https://discuss.pytorch.org/t/is-nomkl-still-recommended-for-omp-issue-on-mac/157388

Ran the commands from an ipython session and got this error:
OMP: Error #15: Initializing libiomp5.dylib, but found libomp.dylib already initialized.
OMP: Hint This means that multiple copies of the OpenMP runtime have been linked into the program. That is dangerous, since it can degrade performance or cause incorrect results. The best thing to do is to ensure that only a single OpenMP runtime is linked into the process, e.g. by avoiding static linking of the OpenMP runtime in any library. As an unsafe, unsupported, undocumented workaround you can set the environment variable KMP_DUPLICATE_LIB_OK=TRUE to allow the program to continue to execute, but that may cause crashes or silently produce incorrect results. For more information, please see [Intel® Product Support](http://www.intel.com/software/products/support/)

**Comment by jeandut** [comment]
Yes very welcome addition indeed ! Should be added to the FAQ at the end of the README.md. Feel free to open a PR.

**Comment by jeandut** [comment]
Closing as #248 was merged
